.github/workflows/spellchecker.yml

```diff
@@ -26,3 +30,15 @@ jobs:
    
       - name: Run spellchecker
         run: yarn run spellcheck || true
+    
+      - name: Install junit2html
+        run: pip install junit2html
+
+      - name: Generate HTML Report
+        run: junit2html spellchecker/report.junit.xml spellchecker/report.html
```

Comment:
In package.json, the spellcheck command is `spellchecker --config spellchecker/config.yml && junit2html spellchecker/report.junit.xml spellchecker/report.html`. We need either remove the junit2html in that command, or update this action step. No need to have both